README.md

```diff
@@ -26,7 +26,22 @@
           "app_category_type": "public.app-category.productivity"
         },
         "linux": {
-          "icon": "resources/linux/<your-project-id>.png"
+          "icon": "resources/linux/<your-project-id>.png",
+          "debian_config": {
+            "section": "Databases",
+            "depends": [
```

Comment:
I think both `depends` and `suggests` here are presently no-ops but I couldn't see how to easily test them as I didn't get any clear errors like the `libXss.so.1` and failure to start Compass on Red Hat. 

Either way needs an upstream fix, see what looks like an incorrect use of `_.defaults` in https://github.com/unindented/electron-installer-debian/blob/e42b311/src/installer.js#L191 

See also COMPASS-590, created as I don't think this should block the other Linux builds changes.